apps/video/style/video.css

```diff
@@ -441,23 +435,23 @@ header .icon:after {
 }
 
 /* video player controls */
-#videoControls {
+.video-controls {
   position: absolute;
-  top: 0;
-  left: 0;
   width: 100%;
-  height: 100%;
   z-index: 10;
   opacity: 1;
-  will-change: opacity;
-  -moz-transition: opacity 0.5s;
+  visibility: visible;
+  will-change: opacity, visibility;
+  transition: visibility 0s, opacity 0.5s;
 }
-#videoControls.hidden {
+.video-controls-hidden .video-controls {
   opacity: 0;
+  visibility: hidden;
+  transition-delay: 0.5s, 0s;
 }
 
-#videoControls.hidden > * {
-  pointer-events: none;
+.video-controls-bottom {
```

Comment:
IMO, it is cleaner to use 'footer .video-controls' here as opposed to introducing another class - 'video-controls-bottom'.